src/main/java/uk/gov/justice/digital/domain/DomainExecutor.java

```diff
@@ -106,12 +114,12 @@ protected void deleteTable(TableIdentifier info) throws DomainExecutorException
         try {
             if (storage.exists(spark, info)) {
                 storage.delete(spark, info);
-                schema.drop(info);
                 storage.endTableUpdates(spark, info);
             } else {
-                throw new DomainExecutorException("Table " + info.getSchema() + "." + info.getTable() +
-                        " doesn't exists");
+                logger.warn("Delete table " + info.getSchema() + "." + info.getTable() + " not executed as table doesn't exist");
```

Comment:
No change needed, just a comment ICYMI that logger calls can also take a simple format string e.g. `logger.warn("Got {} Expected {}", "foo", "bar");`